README.rst

```diff
@@ -154,25 +151,6 @@ will need to be installed. In Ubuntu-based distributions, this can be done as fo
 .. code-block:: console
 
     sudo apt-get install cmake libgtest-dev
-    cd /usr/src/googletest/googletest
-    sudo cmake
-    sudo make
-    sudo cp libgtest* /usr/lib/
-    sudo mkdir /usr/local/lib/googletest
-    sudo ln -s /usr/lib/libgtest.a /usr/local/lib/googletest/libgtest.a
-    sudo ln -s /usr/lib/libgtest_main.a /usr/local/lib/googletest/libgtest_main.a
-
-Alternatively, the latest Googletest release can be installed from source:
-
-.. code-block:: console
-
-    sudo apt install cmake
-    wget -qO - https://github.com/google/googletest/archive/release-1.8.1.tar.gz | tar -xz
-    cmake -D CMAKE_INSTALL_PREFIX:PATH=$HOME/googletest -D CMAKE_BUILD_TYPE=Release googletest-release-1.8.1
-    make install
-
-If installing Googletest from source, make sure that the included headers and
-libraries are available on your include/library paths.
```

Comment:
I would suggest keeping this, just in case the developer is not using Ubuntu/Debian

Reply:
Cool. It's back.